src/main/java/com/onaio/steps/handler/actions/ExportHandler.java

```diff
@@ -183,7 +185,9 @@ public Queue<FileDecorator> saveFiles() throws IOException {
                 row.add(EMPTY_COLUMN);
                 row.add(EMPTY_COLUMN);
                 row.add(EMPTY_COLUMN);
-                row.add(SURVEY_EMPTY_HH);
+                if(household.getStatus() == InterviewStatus.NOT_REACHABLE){
+                    row.add(SURVEY_EMPTY_HH_NOT_REACHABLE);
+                } else row.add(SURVEY_EMPTY_HH);
```

Comment:
Can we make this a Ternary Operation?
```Java
row.add(household.getStatus() == InterviewStatus.NOT_REACHABLE ? SURVEY_EMPTY_HH_NOT_REACHABLE: SURVEY_EMPTY_HH)
```
Or is that too long?

Or use assignment approach which I find cleaner?

```Java
String status = SURVEY_EMPTY_HH;
if (household.getStatus() == InterviewStatus.NOT_REACHABLE) 
   status = SURVEY_EMPTY_HH_NOT_REACHABLE;
row.add(status);
```